playground/backend/internal/environment/environment_service.go

```diff
@@ -179,8 +179,6 @@ func createExecutorConfig(apacheBeamSdk pb.Sdk, configPath string) (*ExecutorCon
 			getEnv(SLF4jKey, defaultSLF4j),
 		}, ":")
 		executorConfig.RunArgs[1] += jars
-	case pb.Sdk_SDK_GO:
```

Comment:
if you removed it from here, how executorConfig will be created?

Reply:
The config is initialized on line 169. In this switch case, the necessary arguments are added to the default config. The executorConfig for GO doesn't need these additional arguments, so I remove this case.

Reply:
can be questions if just to delete the case with Go. Make it clear, return executorConfig

Reply:
We can just leave this case empty and add a comment that Go doesn't need any additional arguments from the config file

Reply:
Done